DESCRIPTION

```diff
@@ -32,7 +31,9 @@ Imports:
     tools,
     tlf (>= 1.4.0),
     vctrs,
-    writexl
+    writexl,
+    stringr,
+    labeling
```

Comment:
Why do we need these two dependencies? 

I see nothing in the code from the current PR that depends on them.

Reply:
Oh, I see. We are using them; they just weren't included as hard deps.